deploy/default_values.yaml

```diff
@@ -10,7 +10,7 @@ spec:
 
   appNavAPI:
     repository: kappnav/apis
-    tag: KAPPNAV_VERSION
```

Comment:
I'm guessing this was changed inadvertently. This needs to stay as "KAPPNAV_VERSION". The build is responsible for replacing this value with the correct tag for the kAppNav release. Please fix.

Reply:
It is strange.  I didn't change it